data-model.sql

```diff
@@ -378,8 +378,8 @@ CREATE TABLE patient (
 	id bigserial not null,
 	full_name varchar(255) not null,
 	friendly_name varchar(128) not null,
-	dob date,
-	ssn varchar(15),
+	dob varchar(100),
+	ssn varchar(9),
```

Comment:
On the off chance a person types in a SSN with dashes, will that fit in 9 characters?